src/App.js

```diff
@@ -227,9 +118,6 @@ const returnLoadedWalletComponent = (Component, props, rest) => {
   // Check version
   if (reduxState.isVersionAllowed === undefined && !isServerScreen) {
     const promise = version.checkApiVersion();
-    promise.then(() => {
-      wallet.localStorageToRedux();
```

Comment:
No need to call it anymore?

Reply:
No need to. The main goal of this method is to save in redux the registered tokens of the user (because it's the data we can't get from the full node).

We are already doing that in `startWallet` method here: `store.dispatch(reloadData({ tokens: dataToken }));`

Reply:
Ok.